lighthouse-core/scripts/chromium-web-tests/webtests/http/tests/devtools/lighthouse/lighthouse-emulate-run.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
these are `third-party/` too?

Reply:
I guess but... We're gonna be rolling from here to CDT from now on.  Kinda weird. But I get it.

Reply:
in other words: these tests will "live" here and be rolled to CDT as we roll Lighthouse to CDT. Given that, where should it live in our directory structure?